src/librustc_mir/transform/const_prop.rs

```diff
@@ -562,9 +446,5 @@ impl<'b, 'a, 'tcx> Visitor<'tcx> for OptimizationFinder<'b, 'a, 'tcx> {
 
 #[derive(Default)]
 struct OptimizationList<'tcx> {
-    const_prop: FxHashMap<Location, Const<'tcx>>,
-    /// Terminators that get their Operand(s) turned into constants.
-    terminators: FxHashMap<BasicBlock, Const<'tcx>>,
     places: FxHashMap<Local, Const<'tcx>>,
```

Comment:
Can you inline this struct now that it's one-field only?

Reply:
done